app/controllers/membership_applications_controller.rb

```diff
@@ -126,20 +138,32 @@ def authorize_membership_application
   end
 
 
-  def new_upload_file(upload_file_param)
-    if upload_file_param['actual_files']
-      upload_file_param['actual_files'].each do |upload_file|
+  def new_file_uploaded(params)
+
+    successful = true
+
+    if params.fetch('uploaded_file', false) && params['uploaded_file'].fetch('actual_files', false)
+
+      params['uploaded_file']['actual_files'].each do |uploaded_file|
```

Comment:
@patmbolger @thesuss 
Let me know if it looks better now.  If so, someone can kindly squash & merge.